media-video/wireplumber/wireplumber-9999.ebuild

```diff
@@ -97,13 +97,6 @@ src_configure() {
 
 src_install() {
```

Comment:
We can drop src_install entirely now as it'll just then pickup meson_src_install.

Reply:
done